### ISSUE TITLE
sudo access for 'jovyan' not provided

### ISSUE THREAD

**Issue by Shellcat-Zero** [dask/dask-docker#27]
It would appear that sudo access has not been provided for the default 'jovyan' user, or credentials are missing in the documentation, but my docker research tells me that this is intentional for security reasons. The bigger issue is how to run the dask docker image granting sudo access, which does not currently seem to be possible as all iterations of `docker run` seem to fail completely with a 'no environment.yml' error:
```
>>> docker run --rm -it -p 8888:8888 -e GRANT_SUDO="yes" --user root daskdev/dask
+ '[' '' ']'
+ '[' -e /opt/app/environment.yml ']'
+ echo 'no environment.yml'
no environment.yml
+ '[' '' ']'
+ '[' '' ']'
```

and I have not been able to find any documentation explaining how to achieve sudo access in the docker-compose.yml file.

**Comment by mrocklin** [comment]
If memory serves to achieve sudo within Docker containers you may need to run the image with elevated privileges.  I don't recall the exact command to do this, but I suspect that a web search or the Docker documentation may come up with something.

**Comment by mariusvniekerk** [comment]
Several of the default base images for containers actually don't even contain the sudo command because its not installed :P

**Comment by Shellcat-Zero** [comment]
Can anyone refer me to the correct documentation to achieve this? I've searched for a very long time and nothing has worked, I'm also still pretty new to Docker.

The proposed solution I've tried for using `sudo` from [Stackoverflow](https://stackoverflow.com/questions/25845538/how-to-use-sudo-inside-a-docker-container) has also failed, such as modifying the Dockerfile with:
```
RUN apt-get update && \
      apt-get -y install sudo

RUN useradd -m docker && echo "docker:docker" | chpasswd && adduser docker sudo
RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```
I then ran `docker-compose build` followed by `docker-compose up` and the docker user was not to be found.
